### PR TITLE
Add 208 notebook

### PR DESCRIPTION
**Pull request by Debskij**

### CODE REVIEW

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 1]
```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Optical Character Recognition\n",
+    "\n",
+    "In this tutorial optical character recognition is presented. This notebook is continuation of [004-hello-detection](../004-hello-detection) notebook.\n",
```
Comment (ryanloney) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:8]:
```suggestion
    "This notebook tutorial demonstrates how to perform optical character recognition. It is a continuation of the [004-hello-detection](../004-hello-detection) notebook, which shows text detection only.\n",
```

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+    "\n",
+    "Now in addition of previously used [horizontal-text-detection-0001](https://docs.openvinotoolkit.org/latest/omz_models_model_horizontal_text_detection_0001.html), [text-recognition-resnet](https://docs.openvinotoolkit.org/latest/omz_models_model_text_recognition_resnet_fc.html) is used."
```
Comment (ryanloney) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:10]:
```suggestion
    "The [horizontal-text-detection-0001](https://docs.openvinotoolkit.org/latest/omz_models_model_horizontal_text_detection_0001.html) and [text-recognition-resnet](https://docs.openvinotoolkit.org/latest/omz_models_model_text_recognition_resnet_fc.html) iare used together for text detection and then text recognition."
```

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+    "            if not path.isfile(f'{download_folder}/{folder_name}/{precision}/{model_name}.{extension}'):\n",
+    "                raise FileNotFoundError\n",
+    "except FileNotFoundError:\n",
+    "    download_command = f\"omz_downloader --name {detection_model_name},{recognition_model_name} --output_dir {download_folder} --precision {precision}\"\n",
+    "    convert_command = f\"omz_converter --name {recognition_model_name} --precisions {precision} --download_dir {download_folder} --output_dir {download_folder}\"\n",
+    "    # Run commands, first download model than convert it to inferable \n",
+    "    !! $download_command\n",
+    "    print('Models downloded')\n",
+    "    # Models are downloaded straight to output folder, we will keep all not used files outside of models directory\n",
+    "    !! $convert_command\n",
```
Comment (ryanloney) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:93]:
```suggestion
    "    ! $convert_command\n",
```

Comment (ryanloney) [reply notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:93]: I want to show the convert output, so please change (and continue to hide the model downloader output)

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Print annotation in text format\n",
+    "\n",
+    "In addition print annotation based on their position, we sort it based on their position. Closer to upper left corner it is, higher it will be prioritised.\n"
```
Comment (ryanloney) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:371]:
```suggestion
    "Print annotations for detected text based on their position in the input image starting from the upper left corner.\n"
```

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 5]
```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Optical Character Recognition\n",
```
Comment (ryanloney) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:6]:
```suggestion
    "# Optical Character Recognition with OpenVINO\n",
```

**Review by ryanloney — CHANGES_REQUESTED** [review]
I added a few comments. Could you please also download the JPG image from this URL as a step in the notebook? I prefer not to duplicate a 288KB file twice in the repo. 

URL: https://github.com/openvinotoolkit/openvino_notebooks/raw/main/notebooks/004-hello-detection/data/intel_rnb.jpg

**Review by ryanloney — APPROVED** [review]

**Comment by raymondlo84** [comment]
We should use the util functions Helena made on downloading files.

def download_file(
    url: PathLike, filename: PathLike = None, directory: PathLike = None, show_progress: bool = True
):
    """
    Download a file from a url and save it to the local filesystem. The file is saved to the
    current directory by default, or to `directory` if specified. If a filename is not given,
    the filename of the URL will be used.

    :param url: URL that points to the file to download
    :param filename: Name of the local file to save. Should point to the name of the file only,
                     not the full path. If None the filename from the url will be used
    :param directory: Directory to save the file to. Will be created if it doesn't exist
                      If None the file will be saved to the current working directory
    :param show_progress: If True, show an IPython ProgressBar.

Also, we shall print out the command and the model downloader is not showing any progress bar. I thought it was crashed and we shall have some ways to show progress there. 
===
    # Run commands, first download model than convert it to inferable 
    !! $download_command
    print('Models downloded')

===

**Comment by raymondlo84** [comment]
plt.imshow(convert_result_to_image(image, resized_image, boxes_with_annotations, conf_labels=True)) 

We may want to size it up so the result is easier to read. The text at the end is very hard to read.

**Comment by ryanloney** [comment]
> plt.imshow(convert_result_to_image(image, resized_image, boxes_with_annotations, conf_labels=True))
> 
> We may want to size it up so the result is easier to read. The text at the end is very hard to read.

@Debskij - can you fix this in a new PR? ^^

**Pull request merged**